src/covid_model_seiir_pipeline/pipeline/postprocessing/task/resample_map.py

```diff
@@ -16,20 +15,32 @@
 from covid_model_seiir_pipeline.pipeline.postprocessing.model import resampling, loaders
 
 
+logger = cli_tools.task_performance_logger
+
+
 def run_resample_map(postprocessing_version: str) -> None:
+    logger.info('Building resampling map', context='setup')
     postprocessing_spec = PostprocessingSpecification.from_path(
         Path(postprocessing_version) / static_vars.POSTPROCESSING_SPECIFICATION_FILE
     )
     workflow_spec = postprocessing_spec.workflow.task_specifications[POSTPROCESSING_JOBS.resample]
     resampling_params = postprocessing_spec.resampling
     data_interface = PostprocessingDataInterface.from_specification(postprocessing_spec)
+
+    logger.info('Loading resampling data', context='read')
```

Comment:
I'd love to know what this "context" argument is doing. I did some quick searching through the documentation but didn't find it.

Reply:
I think it creates groups for which the timings report.